source/loader/layers/sanitizer/asan_buffer.cpp

```diff
@@ -164,7 +164,8 @@ ur_result_t MemBuffer::getHandle(ur_device_handle_t Device, char *&Handle) {
 }
 
 ur_result_t MemBuffer::free() {
-    for (const auto &[_, Ptr] : Allocations) {
```

Comment:
why?

Reply:
AFAIR compiling on jf cluster with g++7 being newest compiler avail there reports warning about "_" as unused variable. These were the only fixes needed to make UR compilable there.

Reply:
We should not worry about warnings in an old compiler. Just don't set `-Werror`.

Reply:
ok, reverted